README.md

```diff
@@ -0,0 +1,10 @@
+How to run tests on your computer:
+
+1) There are two kind of browsers that you can perform tests against.
+   -Chrome
+   -Edge
+   Please choose one of following and download corresponding WebDriver from internet.
+   If you decided to use Chrome put directory with WebDriver to $PATH as env variable. Here is article how to do it:
+   https://jaktestowac.pl/lesson/webdriver-dodanie-do-path/
```

Comment:
Would be nice if linked article was in English.

Also, read how to add hyperlinks to markdown files.